rta/__init__.py

```diff
@@ -39,6 +45,116 @@ def get_ttp_names(os_types=None):
     return names
 
 
+def get_available_tests(print_list=False, os_filter=None):
+    test_names = []
+    test_metadata = []
+
+    for file in list(CURRENT_DIR_PATH.rglob("*.py")):
+        if file.stem not in ("common", "main") and not file.stem.startswith("_"):
+            module = importlib.import_module(f"rta.{file.stem}")
+            if os_filter and os_filter not in module.PLATFORMS:
+                continue
+            test_names.append(file.stem)
+            test_metadata.append({"name": file.stem, "path": file, "platforms": module.PLATFORMS})
```

Comment:
since RTA names will also change, we may want to consider a UUID for RTAs as well, to keep authoritative references